### DISCUSSION TITLE
Governor: Solana and Ethereum Limits

### DISCUSSION THREAD

**Discussion by claudijd** [wormhole-foundation/wormhole#1877] [Governance] [answered]
In the past 24hrs, there has been a significant demand for notional flow from Solana and Ethereum, which is spiking Governor limits, this is to discuss whether those limits should be increased.

Solana: 99% utilization
Ethereum: 80% (but was higher earlier in the day)

**Comment by evan-gray** [comment]
If this has tailed down, perhaps some analytics can be done in the following 24 hours to see if this volume is sustained. It is possible that the limiting will taper off as previous transfers fall out of the 24 hour sliding window.

**Comment by claudijd** [comment]
Based on queue'd flow, expectation is that it will burn off by 10-11am CST tomorrow, so might not be needed, but wanted to open the discussion anyways.

**Comment by evan-gray** [comment, marked as answer]
Thought I'd pop back by to update based on guardian gossip as of this writing:
- $21m available for Solana on the lowest available guardian
- $33m available for Ethereum “”
- $3.5m available for Aptos on the lowest available updated guardian (4 have yet to take the latest patch)
- to my knowledge, no VAA actually had to wait the full 24hrs.
